README.md

```diff
@@ -105,6 +105,88 @@ here.
   - Only unary calls are supported for now.
 
 
+### Bazel
+
+> :exclamation: ***Disclaimer***:
+> Using gRPC-Web in bazel currently requires [rules_closure] with
+  an unmerged patch \(See [#223](https://github.com/grpc/grpc-web/issues/223)\).
+
+Add the following to your WORKSPACE:
+
+```python
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
I see the intention here and I think it's a good idea. But this section seems a bit too heavy compared to the rest of the README. Can we shorten it somehow or take the details out to another page? Like perhaps we can move some of the details to https://github.com/grpc/grpc-web/blob/master/net/grpc/gateway/examples/echo/tutorial.md?